src/modules/flight_mode_manager/tasks/AutoFollowTarget/FlightTaskAutoFollowTarget.hpp

```diff
@@ -105,8 +138,13 @@ class FlightTaskAutoFollowTarget : public FlightTask
 	bool activate(const vehicle_local_position_setpoint_s &last_setpoint) override;
 	bool update() override;
 
+	// Override parameter update function to check when Follow Target properties are changed
+	void updateParams() override;
+
 protected:
```

Comment:
I am not sure exactly what the benefit of using underscore postfix would be:

"My C# team use underscore prefix for private members, as it works very well with Intellisense because while coding you can type underscore and filter out everything but private members."
https://softwareengineering.stackexchange.com/questions/204443/coding-convention-regarding-the-usage-of-underscores

What benefit would the postfix bring, instead of using the suffix?

Reply:
there are arguments in both directions -- e.g. in the article you linked:
>From a c/c++ point of view
Prefix underscores are only safe with lower case variables, and even then double underscores are reserved
Suffix underscores are always safe, but lead to very ugly constructions like member_->pointer or member_[3]

the point here is that we pick a style and keep it consistent. we chose google style, because it is well documented, has many automated tools available already, is quite close to the majority of most recent PX4 activity, and requires less effort (not needing to re-create an entire style guide and go down the rabbit hole of every user's vastly differing opinions on style)